src/h5web/providers/jupyter/api.ts

```diff
@@ -132,10 +127,7 @@ export class JupyterApi implements ProviderAPI {
         ...baseEntity,
         kind,
         type: convertDtype(dtype),
-        shape:
-          dims.length > 0
-            ? { class: HDF5ShapeClass.Simple, dims }
-            : { class: HDF5ShapeClass.Scalar },
+        shape: dims.length > 0 ? dims : [],
```

Comment:
This can be simplified to `shape: dims`, right ? This way, it will also be ready for the support for null shapes that will come in the next release.

Or is there an edge case that I don't get ?

Reply:
😂